eleventy.config.js

```diff
@@ -165,6 +166,72 @@ export default async function (eleventyConfig) {
 	eleventyConfig.amendLibrary("md", (mdLib) => {
 		mdLib.use(markdownItFootnote);
 
+		// Add container support for note types
+		const noteTypes = ["note", "warning", "success", "info"];
+		const noteTypePattern = new RegExp(`^\\[!(${noteTypes.join("|")})\\]\\s*`, "i");
+
+		noteTypes.forEach((type) => {
+			mdLib.use(markdownItContainer, type, {
+				render: function (tokens, idx) {
+					if (tokens[idx].nesting === 1) {
+						// opening tag
+						return `<pob-note type="${type}">\n`;
+					} else {
+						// closing tag
+						return "</pob-note>\n";
+					}
+				},
+			});
+		});
+
+		// Add support for GitLab-style blockquote alerts: > [!TYPE]
+		// Use a core rule to transform blockquotes into pob-note elements
+		mdLib.core.ruler.after("block", "gitlab_alerts", function (state) {
+			const tokens = state.tokens;
+			for (let i = 0; i < tokens.length; i++) {
+				if (tokens[i].type === "blockquote_open") {
+					// Check if the next token is a paragraph containing an alert marker
+					const nextToken = tokens[i + 1];
+					if (nextToken && nextToken.type === "paragraph_open") {
```

Comment:
We can use null chaining, e.g. `if (nextToken?.type === "paragraph_open")`.

You should fix all instances of this, if there are others.